protection.lua

```diff
@@ -119,7 +119,7 @@ minetest.register_node("simple_protection:mark", {
 	tiles = {"simple_protection_marker.png"},
 	groups = {dig_immediate=3, not_in_creative_inventory=1},
 	drop = "",
-	use_texture_alpha = true,
+	use_texture_alpha = "clip",
```

Comment:
Needs backwards compat for older versions. Please check against the corresponding `minetest.features` flag.
For example: `minetest.features.use_texture_alpha_string_modes and "clip" or true`

Reply:
Okay, I'll get this done & update pull request.